packages/pdf/src/RPA/PDF/keywords/model.py

```diff
@@ -32,6 +32,7 @@
 
 
 Coords = Tuple[int, ...]
+BOXES_FLOW_NOT_SET = -999.99
```

Comment:
Would be best to set it to `0.5` as [default](https://pdfminersix.readthedocs.io/en/latest/reference/composable.html#laparams), so we'd keep the behavior consistent. (and set this value no matter what inside as param -> the effect stays the same if the user doesn't provide a different value)